src/file_parsing_uq.jl

```diff
@@ -0,0 +1,507 @@
+using TOML
+using Distributions
+using EnsembleKalmanProcesses.ParameterDistributions
+
+
+# "Public" functions (i.e., functions intended to be called by user):
```

Comment:
I agree we should add this into a module.

Reply:
`file_parsing_uq.jl` had been turned into a module `UQParameters.jl`.